lib/ecom/session.ts

```diff
@@ -0,0 +1,51 @@
+import { createCookieSessionStorage } from '@remix-run/node';
+import { Tokens } from '@wix/sdk';
+import { createApi, createWixClient, getWixClientId } from './api';
+
+export type SessionData = {
+    wixEcomTokens: Tokens;
+    wixClientId: string;
+};
+
+const { getSession, commitSession } = createCookieSessionStorage<SessionData, void>({
+    cookie: {
+        name: '__session',
+        maxAge: 3600 * 24 * 100, // 100 days
+        secure: true,
+        httpOnly: true,
+        sameSite: 'lax',
+
+        secrets: [process.env.SESSION_COOKIE_SIGNING_SECRET ?? 's3cret1'],
+    },
+});
+
+export { commitSession };
+
+export async function initializeSession(request: Request) {
```

Comment:
`initializeEcomSession`, `initializeEcomApi`?

Reply:
There's remix session as well.

Reply:
basically this function initializes ecom session always and remix session if it is not yet initialized